packages/paste-website/src/pages/primitives/tooltip-primitive/index.mdx

```diff
@@ -0,0 +1,219 @@
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {defaultExample, rightExample, styledExample} from '../../../component-examples/TooltipPrimitiveExamples';
+
+export const pageQuery = graphql`
+  {
+    allPastePrimitive(filter: {name: {eq: "@twilio-paste/tooltip-primitive"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/tooltip-primitive/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Tooltip Primitive"
+  categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/primitives/tooltip"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/primitives-tooltip--simple-tooltip"
+  data={props.data.allPastePrimitive.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Tooltip Primitive
+
+The tooltip primitive is an unstyled functional version of a tooltip. It can be used to build a component following the [WAI-ARIA Tooltip](https://www.w3.org/TR/wai-aria-practices/#tooltip)
+pattern. Our [Tooltip](/components/tooltip) is built on top of this primitive.
+
+The purpose of providing these unstyled primitives is to cater for instances when the styled Tooltip provided by Paste doesn’t meet the requirements needed to solve a unique or individual customer problem.
+At that point you are welcome to fall back to this functional primitive to roll your own styled tooltip while still providing a functional and accessible experience to your customers.
+However, we strongly recommend reviewing your design proposal with the Design Systems team before doing so.
+
+This primitive should be used to compose all custom tooltips to ensure accessibility and upgrade paths.
+
+<Callout variant="warning">
+  <CalloutTitle as="h3">Before you roll your own tooltip...</CalloutTitle>
+  <CalloutText>
+    We strongly suggest that all components built on top of this primitive get reviewed by the Design Systems team and
+    go through the UX Review process to ensure an excellent experience for our customers.
+  </CalloutText>
+</Callout>
+
+## Examples
+
+### Basic Tooltip
+
+<LivePreview
+  scope={{
+    useTooltipPrimitiveState,
+    TooltipPrimitive,
+    TooltipPrimitiveReference,
+    Button,
+  }}
+  noInline
+>
+  {defaultExample}
+</LivePreview>
+
+### Controlling placement
+
+The placement of the tooltip is configurable via the placement prop.
+The available placement options are available in the [props description here](#usetooltipprimitivestate).
+
+<LivePreview
+  scope={{
+    useTooltipPrimitiveState,
+    TooltipPrimitive,
+    TooltipPrimitiveReference,
+    Button,
+  }}
+  noInline
+>
+  {rightExample}
+</LivePreview>
+
+---
+
+## Styling a Custom Tooltip
```

Comment:
It's better, but I think we can do a better job connecting the dots. For example by not pulling the `Box` out to a separate component you almost hide what you're doing with creating the styles.

You also don't explain _why_ you need `StyledBase` or where you got it from.

As a consumer of the system I need to get everything I need to know from this page without looking at your code in GitHub.

Reply:
I've made some adjustments. Let me know what you think.